pkg/util/validation/limits_test.go

```diff
@@ -725,7 +725,6 @@ differentuser:
 
 	for name, tt := range tc {
 		t.Run(name, func(t *testing.T) {
-
```

Comment:
spurious change

Reply:
sry, formatter is a bit too invasive - added it back now 👍